tests/helpers.py

```diff
@@ -1111,6 +1111,44 @@ def measure_pvc_deletion_time(interface, pv_name):
     return total.total_seconds()
 
 
+def pod_start_time(pod_obj):
+    """
+
+    Function to measure time taken for container(s) to get into running state
+
+    Args:
+        pod_obj(obj): pod object to measure start time
+
+    Returns:
+        containers_start_time(dict):
+        Returns the name and start time of container(s) in a pod
+
+    """
+    time_format = '%Y-%m-%dT%H:%M:%SZ'
+    containers_start_time = {}
+    start_time = pod_obj.data['status']['startTime']
+    start_time = datetime.datetime.strptime(start_time, time_format)
+    containers_in_pod = len(pod_obj.data['status']['containerStatuses'])
```

Comment:
I suggest to remove (and fix indentation):
```
   containers_in_pod = len(pod_obj.data['status']['containerStatuses'])
   if containers_in_pod == 1:
        started_time = pod_obj.data['status']['containerStatuses'][0][
            'state']['running']['startedAt']
        started_time = datetime.datetime.strptime(started_time, time_format)
        container_name = pod_obj.data['status']['containerStatuses'][0]['name']
        container_start_time = (started_time - start_time).seconds
        containers_start_time[container_name] = container_start_time
    else:
```
The function should work correctly without this even when there is only one container. This should remove unnecessary code duplication.

Reply:
> The function should work correctly without this even when there is only one container. This should remove unnecessary code duplication.

oh yeah! :) I'll fix this.